routetable/route_table.go

```diff
@@ -37,6 +37,13 @@ const (
 	maxConnFailures    = 3
 )
 
+type RTProtocol int
+
+const (
+	RTProtocolUnspec       RTProtocol = 0
+	RTProtocolVXLANManaged RTProtocol = 0x50
```

Comment:
I think we'll want a way to configure the value for this, since 0x50 might already be taken on some systems. 

Probably good enough to add a new FelixConfiguration option. That will require some changes to this PR as well as a new one in libcalico to make it configurable through the datastore.

Reply:
OK I'll add that